include/solo5/mft_abi.h

```diff
@@ -39,11 +39,13 @@
 #define MFT_VERSION 1
 
 /*
- * Supported device types.
+ * Supported manifest entry types.
  */
 typedef enum mft_type {
-    MFT_BLOCK_BASIC,
-    MFT_NET_BASIC
+    MFT_BLOCK_BASIC = 1,
+    MFT_NET_BASIC,
+    MFT_RESERVED_ZERO = (1U << 31),
```

Comment:
Kind of pedantic, but would you mind making this `30` instead of `31` to avoid having to deal with unsigned/signed uncertainties?

Reply:
Sure, will do. Good point since `enum` is signed.